lib/initial-setup/WP_Auth0_InitialSetup.php

```diff
@@ -162,45 +120,26 @@ public function cant_exchange_token_message() {
 
 	public function rejected_message() {
 		?>
-	  <div id="message" class="error">
+	  <div class="notice notice-error">
 		<p>
 		  <strong>
-			<?php echo __( 'The required scoped were rejected.', 'wp-auth0' ); ?>
+			<?php _e( 'The required scoped were rejected.', 'wp-auth0' ); ?>
```

Comment:
No change in this commit, but I assume `scoped` is a typo? Should be `scopes`? If so might as well change now too.

Reply:
```suggestion
			<?php _e( 'The required scopes were rejected.', 'wp-auth0' ); ?>
```